blocks/embed/embed.js

```diff
@@ -0,0 +1,156 @@
+const scPlayerHeight = window.hlx.aemassets?.videosmartcrop?.height || '44rem';
+let isDMVideo = false;
+const dmVideoBehaviour = {
```

Comment:
Can't these parameter be specified in the video smart crop url while authoring itself?